src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private static ArrayList<Task> todoList;
+
+    public TaskList() {
+        todoList = new ArrayList<>();
+    }
+
+    public static ArrayList<Task> getTodoList () {
+        return todoList;
+    }
+
+    public void add (Task task) {
+        todoList.add(task);
+    }
```

Comment:
I like how you named your variables with verbs, easy to understand at one glance.